src/depolarizing-noise.lisp

```diff
@@ -100,3 +101,95 @@ It should be that PX + PY + PZ <= 1.
 ;;; Don't compile things for the depolarizing-qvm.
 (defmethod compile-loaded-program ((qvm depolarizing-qvm))
   qvm)
+
+;;;
+;;; for want of a better idea, we replicate most of the above for stabilizer QVMs
```

Comment:
I'm not a fan of them but we could probably have a mixin class. not necessary for this PR; I'm not especially bothered by the repetition